tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -156,6 +156,11 @@ def get_permissions(self):
         return []
 
     def augment(self, resources):
+        if (self.resource_type.service == "cam"
```

Comment:
this belongs in a subclass for policy rather then the parent having to hardcode exceptions lists for each subclass

Reply:
refactor to use subclass override